### PR TITLE
update code

### PR DESCRIPTION
**Pull request by j-j-gajjar**
- > Update .gitignore
- > Update .pubspec.yaml
- > uopdate code format

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -51,28 +46,14 @@ dev_dependencies:
...
 flutter:
-
   # The following line ensures that the Material Icons font is
   # included with your application, so that you can use the icons in
   # the material Icons class.
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
   assets:
-    - images/bg.jpg
```
Comment (j-j-gajjar) [inline pubspec.yaml:62]: You must be aware that this is a very bad structure. @wahyush04

File: lib/hilmykun_profile.dart [hunk 2]
```diff
@@ -70,14 +69,14 @@ class HilmyProfile extends StatelessWidget {
                 ),
                 Card(
                   color: Colors.white,
-                  margin: EdgeInsets.symmetric(
```
Comment (j-j-gajjar) [inline lib/hilmykun_profile.dart:73]: You must know that it would be best to use the const keyword here. @wahyush04

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # Miscellaneous
```
Comment (j-j-gajjar) [inline .gitignore:1]: I have attached the .gitignore file given in Flutter's repo here @wahyush04

**Review by j-j-gajjar — COMMENTED** [review]
Please verify it, and tell me what is wrong with my PR @wahyush04

**Comment by wahyush04** [comment]
can u fix that problem? and i can merge it to my repo😅

**Comment by j-j-gajjar** [comment]
bro, I have already solved it, and that it is in this PR #17 

Please verify.

**Comment by wahyush04** [comment]
OK bro

**Comment by wahyush04** [comment]
Thx bro

**Pull request merged**